community/salt/APKBUILD

```diff
@@ -1,67 +1,166 @@
+6788663f24c9b4427bb3bd2fac2164b5ed34a0a2eaada0ca50600a9dd1e9892a  salt-api.initd
 383475b21261ac22c5930e99060d53630dfb35aac67aa03a18ec738e0f4dff50  salt-master.confd
-ae9236919c3fee3eef0ef8ad54334d6f833a51bbd4d42c40214614498acbe573  salt-minion.initd
+40c9864ab001c00de60b52baff5c9cf7c2c0f248a1e02fb15260ae70287bce70  salt-master.initd
 286148f5391d42c04a62a13cc125fa2130b5821e50da913c5a20d3a913e5f2d1  salt-minion.confd
+ae9236919c3fee3eef0ef8ad54334d6f833a51bbd4d42c40214614498acbe573  salt-minion.initd
+66a663c426e2fa157ea78f7f9b2f33f17b72dbc48e119f8dd2609aab8f8370e7  salt-syndic.confd
 6a453f63e51f2cc1ecb024ee8e7fa1beab7fbcea010f3eb10ea23bdb2383e7f5  salt-syndic.initd
-66a663c426e2fa157ea78f7f9b2f33f17b72dbc48e119f8dd2609aab8f8370e7  salt-syndic.confd"
-sha512sums="608e2a43bc8c55eeb8c9e0f9207306e591151940ad1ffb869ada12963c0790f37653936e539c0b70439c13180909de2c763db6c30e7f9e2087068985d1b3ff91  salt-2016.3.2.tar.gz
-324e687e8b0eaccb799b4a719448341a4e09cda3add102890beddaf371ac42957bbc1622f145b2b86420061655bb27f27879302b5b34c6eee1987d2dcae6b8ef  salt-master.initd
+df59a9d208d13ce14a8aa1eaed410aeb5f05d8473b606eb98d22a38a30c6e1b2  0001-alpine-support.patch"
+sha512sums="89a6f9e7d8a337ec5addda2374c833d251162150c4e57ce34f7f9e81a75f540539071064c0e7c4486ef6e447c6422363f868e9b035e4c5af2f3d320e98e0226d  salt-2016.3.3.tar.gz
+975ba2f5e681fbd62045da61cc3dc065b148683a07b5df7eca9f131e47314eb6bfa8660ca1c06a3bd93683c7097d0ff9f8e514273dd24d82fb2de6a255e6b275  salt-api.confd
+cf196bd47761b4c22ed3f79815a0ebaa378ebdfc1bb29ee45d2db7c0f99bd65583c59accdb7c34d8cc1a2a33dac999e2e1007f9a8f435ef892ed6ee8895960ff  salt-api.initd
 cfbbeb8023a383e7c42d84e3346edfd068c9ec7650c4ddc3caa38534da325a67497e1f06ca02cc1f0941b7348a3af6d1dca7cd6f2bcb3612ca10e1ec98997e5a  salt-master.confd
-c6634a592c6f3e65dd2b704cb500486bf8004c5b287d4a4d42b09af36ef129c59d6a89f005af058cf7911e8587d927b3db931186569084f13ebaca56f6ef93a0  salt-minion.initd
+324e687e8b0eaccb799b4a719448341a4e09cda3add102890beddaf371ac42957bbc1622f145b2b86420061655bb27f27879302b5b34c6eee1987d2dcae6b8ef  salt-master.initd
 0051e13351cef8db81dc075a194bb384723f07d5591b5b4d3f3adf4180afaf7beced470ab79ceca9d1ec0dae62dbd72084eb76af009fc78411a011050a94a2ed  salt-minion.confd
+c6634a592c6f3e65dd2b704cb500486bf8004c5b287d4a4d42b09af36ef129c59d6a89f005af058cf7911e8587d927b3db931186569084f13ebaca56f6ef93a0  salt-minion.initd
+bafc6ea10cdafd0aef868feb35aecbe4ae6a7dff0ae42862bded85715ad763eb89e1ed27437866a7e5f2b9f7064e3c2a3fb59814487744ba4227238d95cf3818  salt-syndic.confd
 d71133e834685304e0167554035ebbc861252f972bbe981cc71e45b70f15d94a28a02a369463c9a641372919689f96b62a0408b14f824ad986d536e52b1e5ec0  salt-syndic.initd
-bafc6ea10cdafd0aef868feb35aecbe4ae6a7dff0ae42862bded85715ad763eb89e1ed27437866a7e5f2b9f7064e3c2a3fb59814487744ba4227238d95cf3818  salt-syndic.confd"
+2ae35d146ee44154ed8ad910fe0afd9854bf7ded21683e0d06edcae4f194823e172a21147fec5b21c967efcfcf5d1d05cf4696c71fe32cf6ba5c49c4e0b643be  0001-alpine-support.patch"
```

Comment:
APKBUILDs should use tabs for indentation.
Tip: Use [editorconfig](http://editorconfig.org/).